examples/prometheus/README.md

```diff
@@ -165,11 +165,16 @@ builds where the fact they have not started could be cited as resulting from use
 
 NOTE:  OpenShift Online monitors builds in a fashion similar to this today.
 
-> sum(rate(openshift_build_total{phase="Error"}[10m])) / sum((rate(openshift_build_total{phase="Complete"}[10m]) + rate(openshift_build_total{phase="Error"}[10m]))) * 100
+> sum(openshift_build_total{job="kubernetes-apiservers",phase="Error"})/(sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Complete|Error"})) * 100
 
-Calculates the error rate for builds over the last 10 minutes, where the error might indicate issues with the cluster or namespace.  Note, it ignores build in the "Failed" and "Cancelled" phases, as builds typically end up in
+Calculates the error rate for builds, where the error might indicate issues with the cluster or namespace.  Note, it ignores build in the "Failed" and "Cancelled" phases, as builds typically end up in
 one of those phases as the result of a user choice or error.  Administrators after some experience with their cluster could decide what is an acceptable error rate and monitor when it is exceeded.
 
+> ((sum(openshift_build_total{job="kubernetes-apiservers",phase="Complete"})-sum(openshift_build_total{job="kubernetes-apiservers",phase="Complete"} offset 1h))  / (sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Failed|Complete|Error"}) - (sum(openshift_build_total{job="kubernetes-apiservers",phase=~"Failed|Complete|Error"} offset 1h)))) * 100
```

Comment:
I wonder if splitting this across multiple lines would make this more easy to parse for the reader ... something like 
sum(....) -
sum(.... offset 1h) /
sum(.....) - 
sum(.... offset 1h)

Reply:
The renderer is already splitting it:
https://github.com/openshift/origin/blob/4bc82112645058e0c325430244b927e0fbb1d27b/examples/prometheus/README.md

there does appear to be a markdown issue w/ it interpretting the =~ as a strikethrough though.